android/app/src/main/res/drawable/launch_background.xml

```diff
@@ -2,11 +2,14 @@
 <!-- Modify this file to customize your launch splash screen -->
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:drawable="@android:color/white" />
-
+    <!-- android:gravity="center"  -->
```

Comment:
Do not add any extra features/code apart from which should be added to solve related issue.
Do you think this files should be added for post validation ?

Reply:
No it's from previous pull request